frontend/src/App.tsx

```diff
@@ -35,14 +38,14 @@ const App = () => {
               }
             >
               <Switch>
-                <Route exact path="/bookings" component={Booking} />
+                <WithFlagRoute exact path="/bookings" component={Booking} />
                 <Route exact path="/create" component={CreateBooking} />
                 <Route
                   exact
                   path="/directory"
                   component={InterpreterDirectory}
                 />
-                <Redirect to="/bookings" />
+                <Redirect to={checkIfFlag() ? '/create' : '/bookings'} />
```

Comment:
I think /directory will be more appropriate

Reply:
![image](https://user-images.githubusercontent.com/46575401/112376739-0ae06200-8ca2-11eb-8403-4e8d44a453d6.png)

Reply:
@fwpushan  And thanks for good question!